pkg/kubelet/network/kubenet/kubenet_linux.go

```diff
@@ -139,6 +139,7 @@ func (plugin *kubenetNetworkPlugin) Event(name string, details map[string]interf
 		cidr.IP.To4()[3] += 1
 
 		json := fmt.Sprintf(NET_CONFIG_TEMPLATE, BridgeName, plugin.MTU, network.DefaultInterfaceName, podCIDR, cidr.IP.String())
+		glog.V(2).Infof("CNI network config set to %v", json)
```

Comment:
Do you want to print out the entire json blob at V(2) on every event? 
I guess right now there's only 1 event, sugges printing the event type at V(2) and the whole json at V(4).